src/scanoss/agent/Makefile

```diff
@@ -0,0 +1,68 @@
+######################################################################
+# Copyright (C) 2010-2011 Hewlett-Packard Development Company, L.P.
```

Comment:
Please update the copyright information.

Reply:
Solved